packages/lpa-questionnaire-web-app/src/services/pdf.service.js

```diff
@@ -1,66 +1,50 @@
-const fetch = require('node-fetch');
-const uuid = require('uuid');
-
-const config = require('../config');
-const { createDocument } = require('../lib/documents-api-wrapper');
-const { generatePDF } = require('../lib/pdf-api-wrapper');
-const logger = require('../lib/logger');
-
-const FILE_NAME = 'Reply-form';
-
-const getHtmlReply = async (reply) => {
-  const log = logger.child({ replyId: reply.id, uuid: uuid.v4() });
-  /* URL back to this service front-end */
-  const url = `${config.server.host}/${'<html><h1>Simple html file</h1></html>'}/${reply.id}`;
-  let response;
-
-  try {
-    log.info({ url }, 'Generating HTML reply');
-    response = await fetch(url);
-
-    log.debug(
-      {
-        status: response.status,
-        statusText: response.statusText,
-      },
-      'Reply HTML generated'
-    );
-  } catch (err) {
-    log.error({ err }, 'Failed to generate HTML reply');
-    throw err;
-  }
-
-  const ok = response.status === 200;
-
-  if (!ok) {
-    log.error({ status: response.status }, 'HTTP status code not 200');
-    throw new Error(response.statusText);
-  }
-
-  log.info('Successfully generated HTML reply');
-  return response.text();
+const nunjucks = require('nunjucks');
+const path = require('path');
+const checkAnswersSections = require('../lib/check-answers-sections');
+const appealSidebarDetails = require('../lib/appeal-sidebar-details');
+
+// *****CONFIG***** \\
+
+nunjucks.configure([
```

Comment:
This configuration is the same as in the app.js file, shouldn't this be shared?

Reply:
It can be shared when we need to share it, I though it so diminutive that it it was fine as is.

Reply:
OK that's fine